fixtures/pytest_store.py

```diff
@@ -207,3 +207,8 @@ def write_line(line, **kwargs):
         # resume capturing
         with diaper:
             store.capturemanager.resumecapture()
+
+
+def factory(cls, appliance_context=None, *args, **kwargs):
```

Comment:
because those objects break on appliance switches when pushing/poping the appliance stack